File: JanperCab/3-Application/Dtos/DuraformWrapColor/DuraformWrapColorDto.cs
namespace _3_Application.Dtos.DuraformWrapColor
{
    public class DuraformWrapColorDto
    {
        public int Id { get; set; }

        public int DuraformWrapTypeId { get; set; }

        public string Name { get; set; }

        public string ImageUrl { get; set; }

        public bool IsJanperMatching { get; set; }

        public bool IsLaminexMatching { get; set; }
    }
}

File: JanperCab/5-JanperCab/ClientApp/src/app/_models/duraform-wrap-color/DuraformWrapColorDto.ts
import { DuraformWrapTypeDto } from './../duraform-wrap-type/DuraformWrapTypeDto';
export class DuraformWrapColorDto {
  id: number;
  duraformWrapTypeId: number;
  name: string;
  imageUrl: string;
  isJanperMatching: boolean;
  isLaminexMatching: boolean;

  duraformWrapType: DuraformWrapTypeDto;
}
